phaseshifts/backends.py

```
@@ -29,59 +29,155 @@ def autogen_from_input(self, bulk_file, slab_file, tmp_dir=None, **kwargs):
+        raise BackendError(
+            "EEASiSSS did not produce '{}' in '{}'.".format(output_file, workdir)
+        )
+    return [output_path]
+
+
+class EEASiSSSBackend(PhaseShiftBackend):
+    """EEASiSSS backend using native library or ViPErLEED."""
 
     name = "eeasisss"
 
     def autogen_from_input(self, bulk_file, slab_file, tmp_dir=None, **kwargs):
```

Ruff (0.14.10)

Missing type annotation for `**kwargs`

(ANN003)